parameter_store_client_test.go

```diff
@@ -100,6 +113,43 @@ func TestClient_GetParametersByPath(t *testing.T) {
 			path:          "/my-service/dev/",
 			expectedError: errSSM,
 		},
+
+		{
+			name: "Success For Multiple Pages",
```

Comment:
This case is already covered in the first "Success" test case, maybe amend the other test?